guide/spanish/java/variables/index.md

```diff
@@ -17,32 +17,30 @@ Es posible que haya notado que el **tipo de envoltorio** consiste en tipos escri
 Normalmente, puede _declarar_ (es decir, crear) variables según la siguiente sintaxis: < _tipo de datos_ > <nombre de _variable_ >;
 
 ```java
-// Primitive Data Type 
+// Tipo de dato primitivo 
  int i; 
  
- // Reference Data Type 
+ // Referencia al tipo de dato
  Float myFloat; 
 ```
 
 Puede _asignar_ un valor a la variable de forma simultánea cuando lo declara (lo que se denomina _inicialización_ ), o en cualquier lugar del código después de haberlo declarado. El símbolo **\=** se utiliza para el mismo.
 
 ```java
-// Initialise the variable of Primitive Data Type 'int' to store the value 10 
+ // Inicializa la variable primitiva del tipo 'int' para almacenar el valor 10
  int i = 10; 
  double amount = 10.0; 
  boolean isOpen = false; 
  char c = 'a'; // Note the single quotes 
  
- //Variables can also be declared in one statement, and assigned values later. 
+ // Las variables también pueden ser declaradas en una sentencia, y asignarles el valor en otra
```

Comment:
This sentence looks a bit weird to me. Can one of @freeCodeCamp/spanish-language-editors confirm this is okay or suggest a change?

Reply:
I think it is understandable. This is how I would translate it in order to respect more the original:

`//Las variables tambien pueden declararse en una sentencia, asignarse valores más adelante.`

Reply:
Thanks!